package.json

```diff
@@ -43,5 +47,17 @@
                 "path": "./snippets/fsh-snippets.json"
             }
         ]
+    },
+    "scripts": {
+        "compile": "tsc -watch -p ./",
+        "postinstall": "node ./node_modules/vscode/bin/install"
+    },
+    "devDependencies": {
+        "@types/antlr4": "^4.7.2",
+        "@types/node": "^14.14.25",
+        "vscode": "^1.1.37"
```

Comment:
In order for the `npm run compile` script to run, I had to save `typescript` as a dev dependency. Were you able to get this to work without adding it? I also only added it to `devDependencies` because that's where it was in the CIMPL extension.

Reply:
Hmm, that might be because I have typescript installed globally, and that installation got used. But if it's needed, it should be added, so I'll add it.

Reply:
Hmm now that you mention that, I realized I also have it installed globally, so I wonder why it didn't use that also.